src/components/layout/layout.tsx

```diff
@@ -13,11 +13,16 @@ type Props = {
   children: ReactNode;
   title: string;
   subtitle?: string;
+  hideHeader?: boolean;
```

Comment:
One last minor thing: we don't actually need this `hideHeader` flag, you can just use the `<BaseLayout />` for your NotFound page:
```ts
<BaseLayout title="404 Not Found">
  ...
```

Reply:
Nice one